### PR TITLE
[DOC] Update of steps for renewing custom CA

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

**Documentation**

- Updated _Renewing your own CA certificates_ to include a missing step and make the steps clearer
- Updated _Installing your own CA certificates_ to be consistent
- Mentioned possibility of using certificates in PKCS #12 format -- procedures currently describe install/renewal in PRM format

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [x ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc [hunk 1]
```diff
@@ -5,80 +5,104 @@
...
-# Fetch the existing secret
-kubectl get secret _<ca-cert-secret>_ -o 'jsonpath={.data._<ca-certificate>_}' | base64 -d > _<ca-certificate>_
-----
+kubectl get secret _CA-CERTIFICATE-SECRET_ -o 'jsonpath={.data._CA-CERTIFICATE_}' | base64 -d > _CA-CERTIFICATE_
++
+Replace _CA-CERTIFICATE_ with the name of each CA certificate.
 
-. Rename the old `ca.crt` file to `ca_<date>_.crt`, where _<date>_ is the certificate expiry date in the format _<year>_-_<month>_-_<day>_T_<hour>_-_<minute>_-_<second>_Z, for example `ca-2018-09-27T17-32-00Z.crt`. 
+. Rename the old `ca.crt` file as `ca-__DATE__.crt`,
+where _DATE_ is the certificate expiry date in the format _YEAR-MONTH-DAYTHOUR-MINUTE-SECONDZ_.
```
Comment (laidan6000) [inline documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc:52]: Missing underscore - `_Z`?

**Comment by scholzj** [comment]
_Note: This should resolve #3208 _

File: documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc (hunk 1 above)
Comment (PaulRMellor) [reply documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc:52]: No. That's the format: yyyy-mm-dd**T**hh-mm-ss**Z**

Comment (tombentley) [reply documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc:52]: We really need to highlight the T and Z in there. `_YEAR-MONTH-DAYTHOUR-MINUTE-SECONDZ_` is doesn't really do it.

**Pull request merged**